src/components/Comments/ReplyCommentItem.js

```diff
@@ -353,7 +353,7 @@ const ReplyCommentItem = ({
                   paddingRight: 0
                 }
               ]}>
-              <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
+              <IconEn name="block" size={15.02} color={COLORS.gray400} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line is just changing the color of an icon. As long as `COLORS.gray400` is a defined and valid color, this change should not introduce any issues.
```diff
- <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
+ <IconEn name="block" size={15.02} color={COLORS.gray400} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->